### PR TITLE
Explore: UX/UI improvements for pausing and resuming of live tailing

### LINKED ISSUE #18850: Explore: UX/UI improvements for pausing and resuming of live tailing
UX/UI feedback and improvements for [#18836](https://github.com/grafana/grafana/pull/18836).

implementation to-dos:
- [x] margins of buttons, logs list and logs headline within the logs box are all over the place. should look more consistent
- [x] live logging font and line spacing different than the non-live
- [ ] show which rows are new, needs nicer design than gray
- [x] change stop button text to "Exit live mode"

design to-dos:
- [x] show which rows are new, needs nicer design than gray
- [x] find a place to show meta labels:
  - [x] when playing show rate of messages  (`3 lines/s`)
  - [x] when paused, show number of rendered lines 
- [x] need better feedback of what state you are in
  - [x] add a little diode (either in button or next to it) showing state (playing, paused, error) with the paused state pulsing
  - [x] "Last line received" should move to hover title of the diode
- how to enter live? should we have a separate button to enter live? or have a split Run/Live button? 
- [x] exiting live should print the last seen lines in the normal mode (perhaps have to execute query again with range from live view)
  - [x] show a hint that last lines are shown, and that to get recent lines for time range, query has to be run

Postponed:
- deal with consistent order as a user setting
- in non-live mode we do not have the scrollable area for the logs but the entire screen is scrollable. wouldn't it make more sense to have a sticky bar with buttons and Last line received label in the bottom of the screen and take the list out of the box/make the entire screen scrollable?
an example for scrollable full screen live logging can be found here: https://logz.io/blog/logzio-live-tail/

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
This PR includes several UX/UI improvements for pausing and resuming of live tailing:
- [x] Adds margins/paddings to buttons, logs list and logs headline within the logs box for more consistent look
- [x] Fixes live logging font and line spacing as it was different than the non-live
- [x] Adds nicer design when new rows are showed than - **introduces new background colour for dark and light theme** and fading animation (WIP)
- [x] Changes stop button text to "Exit live mode"

**Which issue(s) this PR fixes**:

Fixes #18850

### CODE REVIEW

**Comment by davkal** [comment]
Typography is looking good. 
For the fresh fading, I'd make the fresh background a little more vibrant, and then have the transition quicker than it is now, but starting later, with a slight delay.

**Review by ivanahuckova — COMMENTED** [review]
@davkal 
> For the fresh fading, I'd make the fresh background a little more vibrant, 

We can definitely do something little more vibrant, but when we were trying for example vibrant blue/darker blue - it was really hard on eyes. Do you have any preference on color? 

> and then have the transition quicker than it is now, but starting later, with a slight delay.

Again, no problem to do it at all. Currently it is 2s fade. Would you like something like 0.5s delay and 1.5s/1s fade? We have tested 0.5s delay and 0.5s fade and both @aocenas and @jessover9000 said that it was way too fast.

File: public/app/features/explore/LiveLogs.tsx [hunk 1]
```diff
@@ -20,15 +22,25 @@ const getStyles = (theme: GrafanaTheme) => ({
   logsRowFresh: css`
     label: logs-row-fresh;
     color: ${theme.colors.text};
-    background-color: ${selectThemeVariant({ light: theme.colors.gray6, dark: theme.colors.gray1 }, theme.type)};
+    animation: fade 2s linear 0s 1 normal forwards;
```
Comment (aocenas) [inline public/app/features/explore/LiveLogs.tsx]: Hmm shame we do not have a unique key for the rows we could do something like this:
Screenshot 2019-09-06 at 17 19 27
But right now each row is unmounted on every change so not possible. @kaydelaney did you found out whether we could use labels + nanosecond timestamp as unique key for the rows?

**Review by aocenas — APPROVED** [review]
Looks good to me code wise, do not have a much of a preference regarding the colors so will let that for others.

**Comment by ivanahuckova** [comment]
@davkal @jessover9000 
I've tried #46464A and 1s wait with 0.5s fade. Let me know what do you think. Do you prefer the previous one, or this? Or would you suggest anything else? 🙂

ezgif com-video-to-gif (2)

**Comment by Ijin08** [comment]
@ivanahuckova I would try using something else than linear on the animation. Here maybe ease out makes sense. This would then affect timing. So that might have to be changed.

**Comment by ivanahuckova** [comment]
> Here maybe ease out makes sense. This would then affect timing. So that might have to be changed.

Thanks @Ijin08! I used ease-out with 1s and it looks so much smoother. Thanks for the tip!

File: public/app/features/explore/LiveLogs.tsx [hunk 2]
```diff
@@ -171,7 +187,7 @@ class LiveLogs extends PureComponent<Props, State> {
           </button>
           <button onClick={this.props.stopLive} className={cx('btn btn-inverse', styles.button)}>
             <i className={'fa fa-stop'} />
```
Comment (aocenas) [inline public/app/features/explore/LiveLogs.tsx]: Should we change the icon here? I think as we changed the text from `Stop` -> `Exit live mode` the stop icon makes less sense.

Comment (aocenas) [reply public/app/features/explore/LiveLogs.tsx]: @jessover9000 I think we can use the `fa-times` here (the small `x`) instead as you originally proposed.

**Review by aocenas — APPROVED** [review]

**Pull request merged**